src/UrlParams.test.ts

```diff
@@ -110,8 +110,8 @@ describe("UrlParams", () => {
   });
 
   describe("returnToLobby", () => {
-    it("is true in SPA mode", () => {
-      expect(getUrlParams("?returnToLobby=false").returnToLobby).toBe(true);
+    it("is false in SPA mode", () => {
+      expect(getUrlParams("?returnToLobby=true").returnToLobby).toBe(false);
     });
```

Comment:
Was this test just wrong initially? Or was there a time where we planned to always return to lobby in spa mode?

Reply:
It seems `returnToLobby` was set to a misleading value in SPA mode, because the SPA doesn't actually do anything with the flag. I changed it here: https://github.com/element-hq/element-call/pull/3051/files#diff-d940aba1d89169ffa38dbeac7ec6282e01dbd906fdca5561fed8c0d7ee5b83a0R269